content/docs/custom-stacks/_index.md

```diff
@@ -0,0 +1,33 @@
++++
+title="Custom stacks"
+weight=4
+creatordisplayname = "Andrew Meyer"
+creatoremail = "ameyer@pivotal.io"
+lastmodifierdisplayname = "Andrew Meyer"
+lastmodifieremail = "ameyer@pivotal.io"
++++
+
+A [stack](/docs/using-pack/managing-stacks) provides the buildpack lifecycle with build- and run-time environments (given by a
+build image and a run image). If you're using the `pack` CLI, running `pack suggest-stacks` will display a list of recommended
+stacks that can be used when running `pack create-builder`, along with each stack's associated build and run images.
+
+To create a custom stack for use with `pack create-builder`, the following requirements must be met for the build and run images.
```

Comment:
Instead of a long sentence a simple title "Requirements" and order list would be more easily digestible.

Reply:
The previous paragraph doesn't mention anything about custom stacks, so I probably can't just have a "requirements" header, but I'll see what I can do to shorten it.